redfish-core/lib/account_service.hpp

```diff
@@ -1601,7 +1639,6 @@ inline void triggerUnauthenticatedACFUpload(
 
     if (acf)
     {
-        std::vector<uint8_t> decodedAcf;
         std::optional<std::string> acfFile{};
```

Comment:
Is there any concern here about doing the work to read the acffile in the cases where the user doesn't have the permission to actually upload the file?

I.e. Any worries about a rogue actor being able to cause bmcweb memory problems by calling this with a large file?

I was wondering if the actual parsing/reading of the acffile should be waiting until the user permissions were confirmed. (I realize this is how the original commit was coded. Just something I was wondering about as I was reviewing the overall changes.)

Reply:
In fact, the current code is hard-coded to allow to read ACF file only under "service" user.
For example,

```
L2555:
        // If user is service
        if (username == "service")
        {
...
                triggerUnauthenticatedACFUpload(asyncResp, oem);
                return;
...
        }
```